examples/objective-c/hello_world/ViewController.m

```diff
@@ -37,7 +37,7 @@ - (void)startEnvoy {
   NSLog(@"Starting Envoy...");
   NSError *error;
   EnvoyClientBuilder *builder = [[EnvoyClientBuilder alloc] init];
```

Comment:
Should we change the name of the builder, since what it builds has changed?

Reply:
`StreamClientBuilder` would be fine yeah

Reply:
Updated